splinterd/Cargo.toml

```diff
@@ -116,6 +116,7 @@ authorization-handler-rbac = [
 biome-credentials = ["database", "splinter/biome-credentials"]
 biome-key-management = ["database", "splinter/biome-key-management"]
 biome-profile = ["splinter/biome-profile"]
+circuit-disband = ["splinter/circuit-disband"]
```

Comment:
This should be behind experimental in splintered as well, it should be stabilized at the same time as circuit-disband in libsplinter

Reply:
Updated